Automattic/simplenote-android
# Update Note Sort

### Fix
Update the sort bar visibility and text to close #907.  The sort bar, which was only visible after submitting a search query, is updated to be always visible when searching and not.  The text is updated to match other platforms.  See the table below for details.

Before|After
-|-
Alphabetically, A-Z|Name: A-Z
Alphabetically, Z-A|Name: Z-A
Newest created date|Created: Newest
Oldest created date|Created: Oldest
Newest modified date|Modified: Newest
Oldest modified date|Modified: Oldest

The text updates also affect the **_Sort order_** item in **_Settings_**.  The order of the sort options is the sort bar match the designs.  The order of the sort options in the setting dialog are left in the order they were previously for a couple reasons.  The order is maintained for backwards compatibility with updated apps and the plan is to remove that setting completely at some point.  See the screenshots below for illustration.

![update_note_sort](https://user-images.githubusercontent.com/3827611/105764988-f3d8fa00-5f14-11eb-9f95-33eabeeb400b.png)

### Test
1. Notice sort bar is visible in note list.
2. Tap sort bar.
3. Notice popup menu with all sort options listed as described above.
4. Tap any sort option in popup menu.
5. Notice list sort order matches option selected.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in ac09b7e1 with:
> Added sort bar at top of note list [https://github.com/Automattic/simplenote-android/pull/1284]
